algo-builder-tester/assets/asa.yaml

```diff
@@ -5,8 +5,6 @@ $ART$:
   unitName: "ART"
```

Comment:
our name is ATF

Reply:
Yea. This name is not used in the test code. Feel free to change.

Reply:
fixed https://github.com/Artcoin-Network/artificial-algorand-contract/pull/8/commits/2a1ba4f4e46df7198e3e7a7daba31dd72518bc0b